coffee-shop/src/pages/Details/components/PageFooter/PageFooter.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import styles from "../PageFooter/PageFooter.module.css";
+import Price from "./Price/Price";
+import Button from "../../../../genericComponents/Button/Button";
```

Comment:
you can import the component from index direct like that 
`import Button from "../../../../genericComponents/Button";`

Reply:
done